README.md

```diff
@@ -25,12 +25,16 @@ MAPBOX_TOKEN=???
 AWS_ACCESS_KEY=???
 AWS_SECRET_KEY=???
 POSTGRES_URL=???
+API_ROOT=http://localhost:8080
 ```
 
 * `MAPBOX_TOKEN`: API key provided by [Mapbox](mapbox.com) for mapping.
 * `AWS_ACCESS_KEY` and `AWS_SECRET_KEY`: Required for AWS CLI - see [AWS docs](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html).
 * `POSTGRES_URL`: URL for database to use for the application. This might be `localhost` for development.
 
+If you would like access to our main database, please contact us at `ericboucher@bayesimpact.org`.
```

Comment:
Is this really a good idea? If we (1) commit the schema so people can run their own databases, and (2) offer a hosted version of the app, will that cover most use cases?